### DISCUSSION TITLE
Error pip install yfinance

### DISCUSSION THREAD

**Discussion by tfanggoro** [teamalgoritma/community#752] [Q&A] [answered]
<img width="699" alt="Screenshot 2023-08-21 at 19 23 04" src="https://github.com/teamalgoritma/community/assets/141115378/6617e44c-40ec-47f8-bdc3-c2e49d19ac5b">

gagal pada saat install library yfinance
menggunakan macbook air M2

**Comment by kevbow** [comment, marked as answer]
Dear @tfanggoro,

Bisa coba menggunakan `conda install -c conda-forge lxml` terlebih dahulu lalu install kembali requirements.txt-nya.

**Reply by tfanggoro** [reply]
terima kasih banyak :D
sudah bisa install yfinance nya
